lib/features/mailbox_dashboard/domain/model/spam_report_state.dart

```diff
@@ -0,0 +1,16 @@
+enum SpamReportState {
+  enabled,
+  disabled,
+}
+
+extension SpamReportStateExtension on SpamReportState {
+
+  String get keyValue {
```

Comment:
It is possible to write it directly in `enum`
```suggestion
enum SpamReportState {
  enabled,
  disabled;
  
  String get keyValue {
      switch(this) {
          case SpamReportState.enabled:
            return 'enabled';
          case SpamReportState.disabled:
            return 'disabled';
    }
  }
}
```